python-client/tests/integration/retry_fetch_trace/test_retry_fetch_until.py

```diff
@@ -77,7 +77,8 @@ def trace_header_is_root(tree):
         response = self.iatk.retry_get_trace_tree_until(
             tracing_header=self.xray_trace_header,
             assertion_fn=trace_header_is_root,
-            timeout_seconds=10
+            timeout_seconds=10,
+            fetch_child_traces=False
```

Comment:
I think we could remove `fetch_child_traces` here as we expect this field will be sent to the service side by default.